docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -0,0 +1,190 @@
+---
+title: MUI X v6.18.0 and the latest features before the next major
+description: New components, polished features, better performance and more.
+date: 2023-11-13T00:00:00.000Z
+authors: ['josefreitas']
+card: false
+tags: ['MUI X', 'News']
+---
+
+<div style="max-width:680px;width: 100%; height: 170px; overflow: hidden; margin-bottom: 16px;">
+  <a href="https://github.com/mui/mui-x/releases/tag/v6.18.0">
+    <img src="/static/blog/mui-x-end-v6-features/intro.png" alt="open v6.18.0 release page" style="width: 100%; height: 100%; object-fit: cover; object-position: center;" />
+  </a>
+</div>
+
+We are happy to share the newest MUI X features, marking the release of [MUI X v6.18.0](https://github.com/mui/mui-x/releases/tag/v6.18.0) and sealing v6 with new components and solid functionalities.
+
+As we gear up to develop the next major version, let's review the notable new features introduced since our last [blog post](https://mui.com/blog/mui-x-mid-v6-features/).
```

Comment:
Nitpick: Could probably use a relative link here.
```suggestion
As we gear up to develop the next major version, let's review the notable new features introduced since our last [blog post](/blog/mui-x-mid-v6-features/).
```